proto/staking/v1beta1/msg.proto

```diff
@@ -84,15 +77,11 @@ message MsgSignerUpdate {
             "github.com/maticnetwork/heimdall/types.ValidatorID",
         (gogoproto.customname) = "ID"
     ];
-    google.protobuf.Any new_signer_pub_key = 3
-        [(gogoproto.casttype) =
-             "github.com/maticnetwork/heimdall/types/common.PubKey"];
-    bytes tx_hash = 4
-        [(gogoproto.casttype) =
-             "github.com/maticnetwork/heimdall/types/common.HeimdallHash"];
-    uint64 log_index    = 5;
-    uint64 block_number = 6;
-    uint64 nonce        = 7;
+    string new_signer_pub_key = 3;
```

Comment:
In @jdkanani's PR, he kept it as `google.protobuf.Any`. Needs to check with him.